File: main/src/main/java/net/fwitz/math/main/binary/complex/functions/erf/ErfiPlot.java
package net.fwitz.math.main.binary.complex.functions.erf;

import net.fwitz.math.binary.complex.functions.Erf;
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot;

public class ErfiPlot {
    public static void main(String[] args) {
        new ComplexFunctionPlot("erfi(z)")
                .computeFn(Erf::erfi)
                .domainX(-5, 5)
                .domainY(-5, 5)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/binary/complex/functions/erf/ErfiPlot.kt
package net.fwitz.math.main.binary.complex.functions.erf

import net.fwitz.math.binary.complex.functions.Erf
import net.fwitz.math.plot.binary.complex.ComplexFunctionPlot

object ErfiPlot {
    @JvmStatic
    fun main(args: Array<String>) = ComplexFunctionPlot("erfi(z)")
        .computeFn(Erf::erfi)
        .domainX(-5.0, 5.0)
        .domainY(-5.0, 5.0)
        .render()
}